miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/display/RealMiniAppDisplay.kt

```diff
@@ -30,7 +25,9 @@ internal class RealMiniAppDisplay(
     miniAppMessageBridge: MiniAppMessageBridge
 ) : MiniAppDisplay, WebView(context), WebViewListener {
 
-    private val miniAppDomain = "$appId.$ASSET_DOMAIN_SUFFIX"
+    private val miniAppDomain = "mscheme.$appId"
```

Comment:
I'm still not really sure about this scheme name because it doesn't really communicate what it's for. Is everyone okay with this?

I'm not sure what would be better though. If we use `miniapp.$appId`, then the URL has `miniapp` twice...

Reply:
@ts-nitish-a-jain @ts-leojoseph-r @Climbatize 
We have discussed but are you really okay with this scheme format: `mscheme.MINI_APP_ID://miniapp/index.html`?
Naming is always the hard decision 😭

Reply:
I did not like this naming from the beginning, but it's not that bad. If we decide to keep it like this it's not a big issue.